tribe-ext-settings-import-export.php

```diff
@@ -0,0 +1,370 @@
+					$message .= sprintf( ' <a href="%1$s">%1$s</a>', 'https://wordpress.org/about/requirements/' );
+					$message .= '</p>';
+					tribe_notice( 'PLUGIN_TEXT_DOMAIN' . '-php-version', $message, [ 'type' => 'error' ] );
+				}
+				return false;
+			}
+			return true;
+		}
+
+		/**
+		 * Register the settings page
+		 */
+		function tribe_settings_menu() {
+			//add_options_page( __( 'Sample Settings Import and Export' ), __( 'Sample Settings Import and Export' ), 'manage_options', 'tribe-settings', 'tribe_settings_page' );
+			add_submenu_page( 'edit.php?post_type=tribe_events', __( 'Settings Import / Export', 'PLUGIN_TEXT_DOMAIN' ), __( 'Settings Import / Export', 'PLUGIN_TEXT_DOMAIN' ), 'manage_options', 'tribe_import_export', array( $this, 'tribe_sie_settings_page' ) );
+		}
+
+		/**
+		 * Render the settings page
+		 */
+		function tribe_sie_settings_page() {
+		    ?>
+			<div class="wrap">
+				<h2><?php _e( 'Settings Import / Export', 'PLUGIN_TEXT_DOMAIN' ); ?></h2>
+
+                <?php
+                // Success and error messages
+                if ( ! empty( $_GET['action'] ) ) {
+
+                    if ( $_GET['action'] == 'import_success' ) {
+	                    $msg = __( 'Settings imported.', 'PLUGIN_TEXT_DOMAIN' );
+	                    $notice_class = 'notice-success ';
+                    }
+	                elseif ( $_GET['action'] == 'import_failed' ) {
+		                $msg = __( 'Settings imported.', 'PLUGIN_TEXT_DOMAIN' );
+		                $notice_class = 'notice-error ';
+	                }
+                    elseif ( $_GET['action'] == 'reset_success' ) {
+	                    $msg = __( 'Reset successful.', 'PLUGIN_TEXT_DOMAIN' );
+	                    $notice_class = 'notice-success ';
+                    }
+                    elseif ( $_GET['action'] == 'reset_failed' ) {
+                        $msg = __( 'Settings reset failed.', 'PLUGIN_TEXT_DOMAIN' );
+                        $notice_class = 'notice-error ';
+                        }
+                    elseif ( $_GET['action'] == 'reset_no' ) {
+	                    $msg = __( 'Reset failed. Please enter "reset" in the text field to reset the settings.', 'PLUGIN_TEXT_DOMAIN' );
+	                    $notice_class = 'notice-error ';
+                    }
+
+	                echo '<div class="notice ' . $notice_class . ' is-dismissible"><p>' . $msg . '</p></div>';
+                } ?>
+
+				<div class="metabox-holder">
+					<div class="postbox">
+						<h3><span><?php _e( 'Export Settings', 'PLUGIN_TEXT_DOMAIN' ); ?></span></h3>
+						<div class="inside">
+							<p><?php _e( 'Export the setting of The Events Calendar, Event Tickets and add-ons for this site as a .json file. This allows you to easily import the configuration into another site.', 'PLUGIN_TEXT_DOMAIN' ); ?></p>
```

Comment:
```suggestion
							<p><?php esc_html_e( 'Export the setting of The Events Calendar, Event Tickets and add-ons for this site as a .json file. This allows you to easily import the configuration into another site.', 'PLUGIN_TEXT_DOMAIN' ); ?></p>
```

Looks like a string we could and should escape.